tests/test_0637-setup-tests-for-AwkwardForth.py

```diff
@@ -18,6 +18,7 @@ def test_00(is_forth):
         py = branch.array(interp, library="ak", entry_stop=2)
         assert py[0]["0"][0] == "expskin_FluxUnisim"
         # py[-1] == <STLMap {'expskin_FluxUnisim': [0.944759093019904, 1.0890682745548674, ..., 1.1035170311451232, 0.8873957186284592], ...} at 0x7fbc4c1325e0>
+        assert py.layout.form == interp.awkward_form(branch.file)
```

Comment:
Every test that can read data with `library="ak"` checks the array's final Form with `awkward_form`'s predicted Form, and they're all exactly equal. (These are Forms without `form_keys`.)